tests/jest/ext.wikibase.facetedsearch.test.js

```diff
@@ -1,19 +1,62 @@
+const actual = require( '../../resources/ext.wikibase.facetedsearch.js' );
+
 describe( 'ext.wikibase.facetedsearch.js', () => {
-	// TODO: Add more tests when the JS is more stable
-	const renderedHTML = `
-		<div class="wikibase-faceted-search__facets"></div>
-		<div id="searchText"><input type="text" /></div>
-	`;
+	test( 'Query segements for list facet with no checked items', () => {
+		document.body.innerHTML = `
+			<div class="wikibase-faceted-search__facet-item"><input class="cdx-checkbox__input" type="checkbox" value="Q1"></div>
+			<div class="wikibase-faceted-search__facet-item"><input class="cdx-checkbox__input" type="checkbox" value="Q2"></div>
+			<div class="wikibase-faceted-search__facet-item"><input class="cdx-checkbox__input" type="checkbox" value="Q3"></div>
+		`;
+
+		expect( actual.getListFacetQuerySegments( document.body, 'P1' ) )
```

Comment:
Why are you putting the string in `document.body`, just to get it out of there again?

Reply:
Right, it takes a `HTMLDivElement`, not a string